doc/Dashboard.md

```diff
@@ -0,0 +1,91 @@
+an average higher than two are green.
+
+![Screen shot of visual representation](img/dashboard-visual.png)
+
+To achieve the result shown above requires a set of "stacked" image files (one
+for each step), as well as an overlay image (in this case, the overlay image
+contains all the text).
+
+To build this example, the images used look like this:  
+![Images Used](img/dashboard-visual-instructions.png)
+
+The block was configured to use these images as follows:
+![Screen shot of visual representation rule configuration](img/dashboard-visual-config.png)
+
+Notice that step 2 has turned green due to the default rule "hueRotate 280" for
+any steps with an average above 2, and the first step has a blurred, grey
+background due to the first rule for average values less than 2.
+
+The **Visual Representation Settings** used to define the visual representation
+must be in JSON format. The supported entries are:
+
+* **`"images"`**: A list of image URLs, one per PB block, in the same order as
+  the 'blocks' list (the list of `url_name`s described above). If the images you
+  wish to use are on your computer, first upload them to the course's "Files and
+  Uploads" page. You can then find the URL for each image listed on that page.
+  Note that each URL must start with "http://" or "https://".
+    * All images listed here will be layered on top of each other, and can be
+    colorized, faded, etc. based on the average value of the student's choices
+    for the corresponding group of MCQs.
+* **`"overlay"`**: (Optional) The URL of an image to be drawn on top of the
+  layered images, with no effects applied.
+* **`"background"`**: (Optional) The URL of an image to be drawn behind the
+  layered images, with no effects applied.
+* **`"colorRules"`**: (Optional) The rules used to determine how the student's
+  average choice values affect each layer's image. Each rule is evaluated in
+  order, and the first matching rule is used. Each rule is a JSON object
+  (`{ ... }`) that may consist of any of the following optional entries.
+  * `if`: An optional expression involving `x` (the student's average choice
+    value). If this is true, then this rule will be used. If this is false, the
+    rule will not be used. If no `if` clause is set, the rule will always be
+    used, unless an earlier rule matched first.  
+    Examples:
+      - `"if": "x > 3"`
+      - `"if": "0 < x < 1"`
+      - `"if": "(x > 3) and (x < 6 or x > 100)"`
+  * `hueRotate`: If this rule matches, adjust the color of all parts of the
+    image that have some saturation. In the above example, this is used to
+    change from a turquiose color to a light green. Valid values are between 0
+    (no change) and 360. Note that this effect will *not* affect any parts of
+    the image have no saturation (i.e. are white, black, or any shade of grey).  
+    Examples:
+      - `"hueRotate": "180"`
+      - `"hueRotate": "x * 30"`
```

Comment:
@antoviaque I figured that was the case, but it's tricky because I'm using HTML/CSS to set the colors of the table, but SVG filters to adjust the colors of the image. And so the format required for each is different. Certainly you can figure out the corresponding `hueRotate` degrees value to match any hue you want, but it would take a bit of trial and error. I couldn't think of a better way to implement this that would allow the colors of each to be specified in the same way.

Although, after thinking about this more just now, it may be possible to use the image as a mask, instead of drawing the image directly, in which case the mask could be applied to a rectangle using the same color specification.